src/index_runner/main.py

```diff
@@ -70,6 +60,31 @@ def main():
             print(f'Message content: {val}')
         for receiver in receivers:
             receiver.queue.put(('ws_event', data))
+        # reload config every few iterations?
```

Comment:
I don't think this will work. Inside src/index_runner/es_indexers/main.py, the _CONFIG variable gets set at the top. That variable then gets used to load the sdk indexer names, etc. I doubt this will affect that variable.